admin_guide/managing_networking.adoc

```diff
@@ -1121,21 +1136,32 @@ spec:
       port: 5432
 ----
 
-This will allow routers to access the service, but will allow pods in other users'
-namespaces to access it as well. (In general, this should not be a problem though, since
-those pods could normally just access the service via the public router anyway.)
+This allows routers to access the service, but will also allow pods in other
+users' namespaces to access it as well. This should not be a problem though,
+since those pods could normally access the service by using the public router.
+
+Alternatively, you can create a policy allowing full access from the default namespace, as in the *ovs-multitenant* plugin:
 
-Alternatively, you can create a policy allowing full access from the default namespace, as
-in the *ovs-multitenant* plugin:
+. Add a label to the default namespace.
++
+[IMPORTANT]
+====
+You only need to do this once for the entire cluster. Cluster Administrator role is required to add labels to namesapces.
```

Comment:
s/Cluster Administrator/The cluster administrator
Probably needs a "The" and the majority of references to "cluster administrator" in the docs is lower case